_posts/en/pages/2017-01-01-download.md

```diff
@@ -62,25 +62,37 @@ linux_instructions: "Linux verification instructions"
 snap_instructions: "Snap package verification instructions"
 download_release: "Click the link in the list above to download the release for your platform and wait for the file to finish downloading."
 download_checksums: "Download the list of cryptographic checksums:"
+download_checksums_sigs: "Download the signatures attesting to validity of the checksums:"
 cd_to_downloads: "Open a terminal (command line prompt) and Change Directory (cd) to the folder you use for downloads.  For example:"
 cd_example_linux: "cd Downloads/"
 cd_example_windows: >
   cd %UserProfile%\Downloads
 
 verify_download_checksum: "Verify that the checksum of the release file is listed in the checksums file using the following command:"
 checksum_warning_and_ok: 'In the output produced by the above command, you can safely ignore any warnings and failures, but you must ensure the output lists "$(SHASUMS_OK)" after the name of the release file you downloaded.  For example:'
-obtain_release_key: "Obtain a copy of the release signing key by running the following command:"
+
+obtain_release_key: "Bitcoin releases are signed by a number of individuals, each with a unique public key. In order to recognize the validity of signatures, you must use GPG to load these public keys locally. You can find many developer keys listed in the <a href='https://github.com/bitcoin/bitcoin/tree/master/contrib/builder-keys'>bitcoin/bitcoin repository</a>, which you can then load into your GPG key database. For example, if you saw the line <pre class='highlight'><code>E777299FC265DD04793070EB944D35F9AC3DB76A Michael Ford (fanquake)</code></pre>you could load that key using this command:"
+
+choosing_builders: >
+  It is recommended that you choose a few individuals from this list who you find
+  trustworthy and import their keys as above, or import all the keys per the
+  instructions in the <code>contrib/builder-key</code> README. You will later use their
```

Comment:
Nitpick: I'd prefer to make this a link, but for the URL to be defined outside of the translation string.

Reply:
Done.